src/Microsoft.Health.Dicom.Functions/Migration/MigrationFilesOptions.cs

```diff
@@ -0,0 +1,41 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.Health.Operations.Functions.DurableTask;
+
+namespace Microsoft.Health.Dicom.Functions.Migration;
+
+/// <summary>
+/// Represents the options for a "migration" function.
+/// </summary>
+public class MigrationFilesOptions
+{
+    internal const string SectionName = "MigratingFiles";
+
+    /// <summary>
+    /// Gets or sets the number of DICOM instances processed by a single activity.
+    /// </summary>
+    [Range(1, int.MaxValue)]
+    public int BatchSize { get; set; } = 100;
```

Comment:
You don't need `BatchSize` and `MaxParallelBatches` as config options because these are presumably function inputs

Reply:
No they aren't, right? You use the value from the input which is correct. You shouldn't use these, as a developer may inadvertently change them and affect the execution on running operations. That could potentially crash the long-running operations as it would change the number of activities and/or their expected output. I looked at the use of `_options` and it's correctly only used for getting the concurrency within a single activity

Reply:
yeah you are right. Removed the unwanted properties and config